app/components/visualizer/main-viz/templates/DynamicPieChart.js

```diff
@@ -0,0 +1,134 @@
+// Configurator: a configuration abstraction which will serve
+// as an abstract configuration for common configuration patterns.
+// As of now, it is coupled to the DynamicPieChart, but eventually
+// this should be ripped out into into a more robust abstraction.
```

Comment:
i am starting to think we need a VizComponent base class of some sort to deal with this issue.